### ISSUE TITLE
com.acktie.mobile.android.barcode Module support for 64 bit architecture in android.

### ISSUE THREAD

**Issue by snigdha-n** [IndieSquare/indiesquare-wallet#4]
Hi Team,

We are migrating our application to the latest 7.0.0GA Appcelerator SDK and we are using this module for barcode scanning. The current module does not support 64 bit architecture. Can we get the updated module with 64 bit support and with SDK 7.0.0.GA.

Thanks in advance
Snigdha

**Comment by walkthelot** [comment]
Yes, please! Titanium 7.x support would be incredibly helpful and appreciated.
